File: table.cpp
#include <iostream>
using namespace std;

void table(int n = 11){

    for (int i = 1;i < 11;i++){
        for (int j = 1; j < 11;j++){
            cout <<i*j<<" ";
            }
        cout << "\n";    
        }
    cout <<"\n";
    }

int main(){
    table();
    return 0;
    }

File: table.py
def table(n):
    for i in range(1,n):
        print("\033[91mThis is a different color")
        for j in range(1,n):
            print(i*j,end = " ")
        print('\n')


if __name__ == '__main__':
    n = int(input('Enter integer = '))
    table(n)
